products/kubernetes-csi-node-driver-registrar.md

```diff
@@ -0,0 +1,105 @@
+---
+title: kubernetes-csi-node-driver-registrar
+category: app
+tags: kubernetes
```

Comment:
we avoided the `kubernetes` tag till now, because `cncf` fit most of the products better, but this seems like a good product to start this with. https://endoflife.date/tags/cncf